index.js

```diff
@@ -417,12 +440,25 @@ Cartero.prototype.processMains = function( callback ) {
 				if( theUrl.charAt( 0 ) === '/' ) return match;
 				if( theUrl.indexOf( 'data:' ) === 0 ) return match; // data url, don't mess with this
 
+				var assetUrl = path.resolve( path.dirname( file ), theUrl );
+				var newAssetName = _this.assetMap[ assetUrl ];
```

Comment:
`assetUrl` here actually takes the value of the absolute path of the asset, since `file` is the absolute path of the css file and `theUrl` is a relative url/path. So I think we should call this variable `absoluteAssetPath` or something like that.